aiourlshortener/shorteners/base.py

```diff
@@ -8,8 +10,10 @@ class BaseShortener(object):
     Base class for all Shorteners
     """
     api_url = None
+    _session = None
 
     def __init__(self, **kwargs):
+        atexit.register(self.close())
```

Comment:
`atexit.register` expects a callable and any arguments
Therefore your call to `atexit.register` should look like:
`atexit.register(self.close)`

Reply:
whoops, you are right. But the schedules callback does not fix this issue as well.